mappings/net/minecraft/village/raid/Raid.mapping

```diff
@@ -13,11 +13,11 @@ CLASS net/minecraft/unmapped/C_szefbyex net/minecraft/village/raid/Raid
 	FIELD f_fnxkdmas DEFEAT_TITLE Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_gknmsuhn SPAWN_ATTEMPTS I
 	FIELD f_jakhyora MAX_ACTIVE_TICKS I
-	FIELD f_jbpzyzdv VICTORY_SUFFIX_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_jbpzyzdv VICTORY Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_kdhitzkh RAID_REMOVE_THRESHOLD_SQUARED I
 	FIELD f_kujahjos started Z
 	FIELD f_kyzgcvgr random Lnet/minecraft/unmapped/C_rlomrsco;
-	FIELD f_lqyogwfr DEFEAT_SUFFIX_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_lqyogwfr DEFEAT_SUFFIX Lnet/minecraft/unmapped/C_rdaqiwdt;
```

Comment:
```suggestion
	FIELD f_lqyogwfr DEFEAT Lnet/minecraft/unmapped/C_rdaqiwdt;
```
The format of this should definitely match `VICTORY` (whether they both have `_SUFFIX` or not) because their translation keys follow the same pattern, and I'm in favor of dropping the `_SUFFIX` suffix to follow the convention of matching the translation key.